paymentsheet/src/test/java/com/stripe/android/paymentsheet/model/PaymentSelectionTest.kt

```diff
@@ -0,0 +1,111 @@
+package com.stripe.android.paymentsheet.model
+
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import com.google.common.truth.Truth.assertThat
+import com.stripe.android.testing.PaymentMethodFactory
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.kotlin.mock
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class PaymentSelectionTest {
+
+    private val context: Context = ApplicationProvider.getApplicationContext()
+
+    @Test
+    fun `Doesn't display a mandate for Link`() {
+        val link = PaymentSelection.Link
+        val result = link.mandateText(
+            context = context,
+            merchantName = "Merchant",
+            isSaveForFutureUseSelected = listOf(true, false).random(),
```

Comment:
Can we write a helper that runs each scenario instead? Randomly passing/failing is a bit sad.